sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -296,6 +296,20 @@ public static String getNonIdempotentWriteRetryPolicy() {
         return System.getenv(NON_IDEMPOTENT_WRITE_RETRY_POLICY_VARIABLE);
     }
 
+    public static int getDefaultHttpPoolSize() {
```

Comment:
In GatewayConnectionConfig, there is a public method `setMaxConnectionPoolSize`,  so I do not think we need to expose changing through System property since the config is exposed in spark, so from spark connector side, we can just call `setMaxConnectionPoolSize`?

Reply:
Yeah - doing it there already. But I want to have more flexibility on this in the future and allow customers to override this without code changes as well.

Reply:
gotcha